apps/server/src/modules/task/controller/task.controller.ts

```diff
@@ -69,4 +68,13 @@ export class TaskController {
 
 		return response;
 	}
+
+	@Delete(':id')
+	async delete(
+		@Param('id', ParseObjectIdPipe) taskId: string,
+		@CurrentUser() currentUser: ICurrentUser
+	): Promise<string> {
+		await this.taskUc.delete(currentUser.userId, taskId);
+		return 'deleted';
```

Comment:
```suggestion
		return true;
```
I think, that is better. And move to uc